tests/qc_checkers/test_reads_per_sample.py

```diff
@@ -69,8 +69,9 @@ def test_reads_per_sample(qc_data_and_exp_val):
         warning_threshold=60,
     )
 
-    assert len(qc_reports) == 4
-    for qc_report in filter(None, qc_reports):
```

Comment:
Ah yes, now I remember, I assumed that it was expected that QC reports could be None :facepalm: